docs/src/components/Layout/index.tsx

```diff
@@ -67,7 +67,7 @@ export default function Page({
 
   return (
     <>
-      <main className="docs-content">
+      <div className="docs-content">
```

Comment:
We can  make this a div now, and in _app.page.tsx change to using main since we no longer have a sidebar (and home page needs a main landmark)